watertap/unit_models/electrodialysis_1D.py

```diff
@@ -670,17 +683,12 @@ def build(self):
             units=pyunits.dimensionless,
             doc="water recovery ratio calculated by mass",
         )
-        self.velocity_diluate = Var(
-            self.flowsheet().time,
-            self.diluate.length_domain,
-            initialize=0.01,
-            units=pyunits.meter * pyunits.second**-1,
-            doc="Linear velocity of flow",
-        )
-        self.velocity_concentrate = Var(
+        self.velocity = Var(
+            self.flow_channel_set,
```

Comment:
Subtle thing, but maybe the typical order should remain (i.e., time first, then space). I'd put flow_channel_set index after time, either between time and length domain, or right after length domain.

Reply:
Really more of a "consistency across models" issue but don't want to hold this up if there's good reason for this

Reply:
@hunterbarber , if you do not like the original naming/treatment, a better design would be put this var under each of diluate and concentrate block rather than introducing an extra 2-member set that is only used once.